entropy/entropy.go

```diff
@@ -0,0 +1,28 @@
+package entropy
+
+import (
+	"crypto/rand"
+)
+
+// CustomRandomSource is a generic interface that any drand node operator can implement
+// to provide their own source of randomness in function GetRandom.
+type CustomRandomnSource interface {
```

Comment:
A last nitpick: let's try to avoid long Java-style name if possible and prefer concise names more in the Go spirit. I have one proposition: What about `Source` ? Since we are in the package `entropy` so that would result in a `entropy.Source` interface. What do you think ? Open to other propositions :)

Reply:
This sounds good! I can make it EntropySource (more concise and still clearly communicates the purpose).